tileserver/__init__.py

```diff
@@ -433,7 +434,13 @@ def read_tile(self, coord, offset=None):
         else:
             fmt = json_format
 
-        raw_data = self.store.read_tile(coord, fmt, 'all')
+        try:
+            raw_data = self.store.read_tile(coord, fmt, 'all')
+        except BotoServerError:
```

Comment:
Yea... I was waffling on that. It probably makes sense as long as we're logging. I'll make that change.